spec/lang/types.md

```diff
@@ -105,14 +105,15 @@ pub enum Discriminator {
     Known(Int),
     /// Tag decoding failed, there is no valid discriminant.
     Invalid,
-    /// We don't know the discriminant, so we branch on the value of a specific byte.
+    /// We don't know the discriminant, so we branch on the value of a specific value.
     /// The fallback keeps the representation more compact, as we often are only
-    /// interested in a couple of values and we don't want to always have 256 branches.
+    /// interested in a couple of values and don't want to always have > 256 branches.
```

Comment:
I think with the int branching, we can just remove this last sentence. The non-fallback representation gets entirely ludicrous for types bigger than a byte.